BenchPress/Generators/ResourceTypes/ApiManagementPolicy.cs

```diff
@@ -0,0 +1,24 @@
+namespace Generators.ResourceTypes;
+
+public class ApiManagementPolicy : ResourceType
+{
+    public ApiManagementPolicy() { }
+
+    public override string Id => "Microsoft.ApiManagement/service/apis/policies";
+    public override string FullName => Id;
+    public override string FriendlyName => "API Management Service Policies";
+    public override string Prefix => "amspol";
+    public override string FunctionPrefix => "ApiManagementPolicy";
+
+    public override IEnumerable<KeyValuePair<string, object>> GetResourceParameters(TestMetadata m)
+    {
+        return new[]
+        {
+            Param("ResourceType", "ApiManagementPolicy"),
+            Param("ResourceName", m.ResourceName),
+            Param("ResourceGroupName", m.ExtraProperties["resourceGroup"]),
+            Param("ServiceName", m.ExtraProperties["serviceName"]),
+            Param("ApiId", m.ExtraProperties["apiName"])
```

Comment:
Similar comment. If "parent"/dependsOn resources are of type "Microsoft.ApiManagement/service/apis", should the keys be `service` and `apis`?

Reply:
Same, it's `apiName` in the generated ARM template.